src/ucp/core/ucp_request.c

```diff
@@ -24,6 +24,31 @@
 
 const ucp_request_param_t ucp_request_null_param = { .op_attr_mask = 0 };
 
+static const char *ucp_request_flag_names[] = {
+    [ucs_ilog2(UCP_REQUEST_FLAG_COMPLETED)]             = "cpml",
+    [ucs_ilog2(UCP_REQUEST_FLAG_RELEASED)]              = "rls",
+    [ucs_ilog2(UCP_REQUEST_FLAG_PROTO_SEND)]            = "proto",
+    [ucs_ilog2(UCP_REQUEST_FLAG_SYNC_LOCAL_COMPLETED)]  = "loc_cmpl",
+    [ucs_ilog2(UCP_REQUEST_FLAG_SYNC_REMOTE_COMPLETED)] = "rm_cmpl",
+    [ucs_ilog2(UCP_REQUEST_FLAG_CALLBACK)]              = "cb",
+    [ucs_ilog2(UCP_REQUEST_FLAG_PROTO_INITIALIZED)]     = "init",
+    [ucs_ilog2(UCP_REQUEST_FLAG_SYNC)]                  = "sync",
+    [ucs_ilog2(UCP_REQUEST_FLAG_OFFLOADED)]             = "offld",
+    [ucs_ilog2(UCP_REQUEST_FLAG_BLOCK_OFFLOAD)]         = "blk_offld",
+    [ucs_ilog2(UCP_REQUEST_FLAG_STREAM_RECV_WAITALL)]   = "strm_r_wtall",
+    [ucs_ilog2(UCP_REQUEST_FLAG_SEND_AM)]               = "snd_am",
+    [ucs_ilog2(UCP_REQUEST_FLAG_SEND_TAG)]              = "snd_tag",
+    [ucs_ilog2(UCP_REQUEST_FLAG_RNDV_FRAG)]             = "rndv_fr",
+    [ucs_ilog2(UCP_REQUEST_FLAG_RECV_AM)]               = "rcv_am",
+    [ucs_ilog2(UCP_REQUEST_FLAG_RECV_TAG)]              = "rcv_tag",
+    [ucs_ilog2(UCP_REQUEST_FLAG_RKEY_INUSE)]            = "rk_use",
```

Comment:
Need to add: `UCP_REQUEST_FLAG_USER_HEADER_COPIED`, `UCP_REQUEST_FLAG_USER_MEMH`, `UCP_REQUEST_FLAG_PROTO_AMO_PACKED`